kubernetes/deployment-production.tmpl

```diff
@@ -133,9 +133,11 @@ data:
   PROJECT_REQUEST_BCC: i2h6y2f6a7o0x7i9@zooniverse.slack.com
   REDIS_URL: 'redis://panoptes-production-redis:6379/0'
   SIDEKIQ_CONCURRENCY: '8'
-  STORAGE_ADAPTER: aws
+  STORAGE_ADAPTER: azure
```

Comment:
Makes sense, I'll switch `STORAGE_ADAPTER` back to aws, but I'll leave these azure ENV vars here because we do need to remember to include them for when we switch over, no harm having them here unless there's a good reason not to @camallen let me know if you disagree.